NOMIS/nomis.go

```diff
@@ -109,13 +116,32 @@ func main() {
 		return
 	}
 
-	for index0, _ := range res.Structure.Keyfamilies.Keyfamily {
+	for index0 := range res.Structure.Keyfamilies.Keyfamily {
+		var annotations = res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation
 		censusEditionData := models.EditionUpdate{}
+		censusInstances := models.Version{}
 		mapData := models.Dataset{}
-		cenId := res.Structure.Keyfamilies.Keyfamily[index0].ID
-		mapData.Title = res.Structure.Keyfamilies.Keyfamily[index0].Name.Value
-		mapData.ID = cenId
+		var cenId string
+		for censusId := range annotations {
+			annoIndex := res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[censusId]
+			switch annoIndex.Title {
```

Comment:
I think single case switch statements should be `If` statements. (Switch statements should contain a default too but don't think this is the place for a switch statement).